pkcs8/src/identifier/object.rs

```diff
@@ -0,0 +1,50 @@
+//! ISO/IEC Object Identifiers (OIDs)
+// TODO(tarcieri): switch to `const-oid`
```

Comment:
This PR is pretty much complete aside from this TODO.

I'd like to add basic OID parsing functionality to the `const-oid` crate (probably as a separate PR) and then use `const_oid::ObjectIdentifier` as the OID type, effectively throwing away everything in this file as it's mostly just a placeholder.